.vscode/settings.json

```diff
@@ -1,8 +1,8 @@
 {
   "editor.tabSize": 2,
-  "vetur.format.options.useTabs": false,
+  "eslint.lintTask.enable": true,
```

Comment:
Made some changes here to align with some of our standards. wrapLineLength and rules set to 110 so we know where our end of line is.